src/discordRPC.js

```diff
@@ -0,0 +1,46 @@
+const rpc = require('discord-rpc');
+const config = {
+	// Joey pls upload the assets to (https://discord.com/developers/applications -> Rich Presence -> Art Assets)
+	appId: "1015217737724874803", //ID
+	details: "ChaiNNing something cool!", //Large text under ChaiNNer
```

Comment:
Do most apps start with a message like this? Or do they just have a blank message unless something is actually happening?

Reply:
You don't have to set anything at all, it can run without them just fine. You could technically start it without an explicit largeImage too, but then it defaults to whatever you have set on the Discord Devportal. This is the most barebones version that runnable:
![image](https://user-images.githubusercontent.com/86712892/188288623-76e5c000-d1c3-4008-bd71-c9303c1d5b79.png)